pages/Hire.vue

```diff
@@ -1,54 +1,120 @@
 <template>
-    <div :style="{ backgroundColor: color }">
-        <v-radio-group v-model="color">
-            <v-radio label="Red" value="Red"></v-radio>
-            <v-radio label="Green" value="Green"></v-radio>
-            <v-radio label="Blue" value="Blue"></v-radio>
-            <v-radio label="White" value="White"></v-radio>
-            <v-radio label="Black" value="Black"></v-radio>
-            <v-radio label="FFBB00" value="#FFBB00"></v-radio>
-            <v-radio label="ABCDEF" value="#ABCDEF"></v-radio>
-            <v-radio label="123456" value="#123456"></v-radio>
-        </v-radio-group>
-        <div class="square" :class="classBindingObject"></div>
-    </div>
+    <v-container
+        class="pa-0"
+        fluid
+        style="
```

Comment:
pleae remove custom styles leftover from wordpress and use vuetify only

Reply:
done